packages/react-core/src/components/Dropdown/DropdownWithContext.tsx

```diff
@@ -126,6 +127,7 @@ export class DropdownWithContext extends React.Component<DropdownProps & OUIAPro
                 direction === DropdownDirection.up && styles.modifiers.top,
                 position === DropdownPosition.right && styles.modifiers.alignRight,
                 isOpen && styles.modifiers.expanded,
+                isFullHeight && styles.modifiers.fullHeight,
```

Comment:
Are we going to get any warnings in consoles about props spreading without any extra checks here before passing isFullHeight to BaseComponent?

Reply:
I should be catching `isFullHeight` above (line 69) so it shouldn't be passed to `BaseComponent` via `props`.

Reply:
oh yeah - i see that now. In hind sight this was a silly question. this is in the css function... Mondays....